packages/url_launcher/url_launcher/README.md

```diff
@@ -96,15 +96,15 @@ takes a string argument containing a URL. This URL
 can be formatted using a number of different URL schemes. The supported
 URL schemes depend on the underlying platform and installed apps.
 
-Common schemes supported:
+Commonly used schemes include:
 
-| Scheme | Action |
-|:---|:---|
-| `http:<URL>` , `https:<URL>`, e.g `http://flutter.dev` | Open URL in the default browser |
-| `mailto:<email address>?subject=<subject>&body=<body>`, e.g `mailto:smith@example.org?subject=News&body=New%20plugin` | Create email to <email address> in the default email app |
-| `tel:<phone number>`, e.g `tel:+1 555 010 999` | Make a phone call to <phone number> using the default phone app |
-| `sms:<phone number>`, e.g `sms:5550101234` | Send an SMS message to <phone number> using the default messaging app |
-| `file:/<path>`, e.g `file:/home` | Opens file or folder using default app association, supported on desktop platforms |
+| Scheme | Example | Action |
+|:---|:---|:---|
+| `https:<URL>` | `https://flutter.dev` | Open URL in the default browser |
+| `mailto:<email address>?subject=<subject>&body=<body>` | `mailto:smith@example.org?subject=News&body=New%20plugin` | Create email to <email address> in the default email app |
+| `tel:<phone number>` | `tel:+1 555 010 999` | Make a phone call to <phone number> using the default phone app |
```

Comment:
Sorry, one more nit since you are touching these lines: I just noticed that this example is actively wrong (this isn't a valid URL because of the spaces). We should fix it while we are here to `tel:+1-555-010-999`.

Reply:
No, problem. Updated tel just like you suggested.